services/custom-forms.ts

```diff
@@ -127,11 +176,69 @@ export async function updateField(
     linkedFormTypeTtl = `${escaped.fieldUri} ext:linkedFormType ${linkedFormTypeUri} .`;
   }
 
+  const libraryEntryData = await getLibraryEntryForField(
+    description.field,
+    description.libraryEntryUri,
+  );
+  let fieldPath = null;
+  let libraryEntryInsertTtl = '';
+  let removeFieldPathWhen = '';
+  let insertFieldPathWhen = '';
+  if (libraryEntryData) {
+    fieldPath = libraryEntryData.fieldPath;
+    const _escaped = {
```

Comment:
we now have `escaped` and `_escaped` this is very confusing when reading the code. What is the difference between one and the other? can we give them a meaningful name?